app/User.php

```diff
@@ -52,4 +52,9 @@ public function updateProfile($request)
         
         return $this->save();
     }
+    
+    public function loggedIn()
+    {
+        return (auth()->user() == $this);
```

Comment:
what's the difference of this function to ```Auth::check()``` in laravel. and i think also middleware is also good way to handle authentication regarding what page should be accessed by user.